### DISCUSSION TITLE
Spain Spanish and Catalan languages

### DISCUSSION THREAD

**Discussion by berni0057** [geovens/gInk#222] [Show and tell]
I've been translated gInk to Spanish (Spain) and Catalan. I am new to GitHub, and I don't know other way to send the files, so I put them in here. Thanks.
[ca.txt](https://github.com/user-attachments/files/18218640/ca.txt)
[es-es.txt](https://github.com/user-attachments/files/18218641/es-es.txt)

**Comment by geovens** [comment]
Thank you!
